### PR TITLE
Notifications borrowing

### PR DESCRIPTION
**Pull request by NazarHladaniuk**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+TELEGRAM_CHAT_ID=
+TELEGRAM_BOT_TOKEN=
```
Comment (yura-hudzovskyi) [inline .env:2]: add blank

File: .env.sample [hunk 2]
```diff
@@ -0,0 +1,2 @@
+TELEGRAM_CHAT_ID=TELEGRAM_CHAT_ID
+TELEGRAM_BOT_TOKEN=TELEGRAM_BOT_TOKEN
```
Comment (yura-hudzovskyi) [inline .env.sample:2]: add blank

**Review by yura-hudzovskyi — CHANGES_REQUESTED** [review]
Seems good, but fix blanks in files

File: .env (hunk 1 above)
Comment (yura-hudzovskyi) [reply .env:2]: And according to the best practice, it will be better if you delete  .env file from git

**Review by yura-hudzovskyi — APPROVED** [review]

**Review by DavidGlusch — APPROVED** [review]

**Pull request merged**